docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/07_Inserting_Media.md

```diff
@@ -38,7 +38,10 @@ You can upload your video to one of the supported media services (many of the [s
 9. Click the button ***Publish*** to make your changes live.
 
 ## Media metadata caching
-Metadata such as the media title or dimensions from the media service may be cached (saved) on your website so that it loads faster for your website users.  If you change something on the media server and you need to clear the cached metadata on your website, then simply re-save or publish a page with the embedded media will clear the website cache of metadata related to this particular piece of media on all pages.
+
+Metadata such as the media title or dimensions from the media service may be cached (saved) on your website so that it loads faster for visitors to your website.
```

Comment:
Remove one of the 'your website' mentions so it's not duplicated and reads a little clearer. Something like:

```suggestion
Metadata such as the media title or dimensions from the media service may be cached (saved) on your website so that it loads faster for your visitors.
```